src/main/java/baseball/util/generator/HintGenerator.java

```diff
@@ -0,0 +1,30 @@
+package baseball.util.generator;
+import baseball.model.Numbers;
+
+import java.util.List;
+public class HintGenerator {
+    public String generateHint(Numbers userNumbers, Numbers computerNumbers) {
+        List<Integer> userNumberList = userNumbers.getNumbers();
+        List<Integer> computerNumberList = computerNumbers.getNumbers();
+        int balls = 0;
+        int strikes = 0;
+        for (int i = 0; i < userNumberList.size(); i++) {
+            if (userNumberList.get(i).equals(computerNumberList.get(i))) {
+                strikes++;
+            } else if (computerNumberList.contains(userNumberList.get(i))) {
+                balls++;
+            }
+        }
+        return generateHintMessage(balls, strikes);
+    }
+    private String generateHintMessage(int balls,int strikes){
+        StringBuilder hint = new StringBuilder();
+        if(balls>0)
+            hint.append(balls + "볼 ");
+        if(strikes>0)
+            hint.append(strikes + "스트라이크");
+        if(balls==0&&strikes==0)
+            hint.append("낫싱");
```

Comment:
```suggestion
        if(balls > 0)
            hint.append(balls + "볼 ");
        if(strikes > 0)
            hint.append(strikes + "스트라이크");
        if(balls == 0 && strikes == 0)
            hint.append("낫싱");
```

이건 진짜 사소한건데... if문이나 for문 안에 띄어쓰기를 이렇게 하는 것을 좋아하긴합니다 ㅋㅋㅋㅋㅋ
이건 진짜 취향 차이라서 그냥 넘기셔도 됩니다!